modules/iam-assumable-role/variables.tf

```diff
@@ -83,11 +83,20 @@ variable "custom_role_policy_arns" {
 }
 
 variable "custom_role_trust_policy" {
-  description = "A custom role trust policy"
+  description = "A custom role trust policy. (Only valid if create_custom_role_trust_policy = true)"
   type        = string
   default     = ""
 }
 
+variable "create_custom_role_trust_policy" {
+  description = <<EOF
```

Comment:
Please write it in one line like "Whether to create a custom role trust policy".

Reply:
4be6b92547033b41acfc231cf18658ba866bf1cb